backend/webapi/Features/ClientLogs/Create.cs

```diff
@@ -0,0 +1,54 @@
+namespace Pidp.Features.ClientLogs;
+
+using FluentValidation;
+using Pidp.Data;
+using Pidp.Models;
+
+public class Create
+{
+    public class Command : ICommand
+    {
+        public int Id { get; set; }
+
+        public string Message { get; set; } = string.Empty;
+
+        public LogLevel? LogType { get; set; }
+
+        public string BrowserInformation { get; set; } = string.Empty;
+    }
+
+    public class CommandValidator : AbstractValidator<Command>
+    {
+        public CommandValidator(IHttpContextAccessor accessor)
+        {
+            var user = accessor?.HttpContext?.User;
+
+            this.RuleFor(x => x.Id).NotEmpty();
+            this.RuleFor(x => x.LogType).NotEmpty();
```

Comment:
There is a weird discrepancy here between the `ClientLog` model, the `Command` and the validator.
The Model and `Command` allow `LogType` to be null, while the validator says that it cannot be null.
These should all be the same.

Reply:
Its up to you if they should all be null or non-null. I think non-null is fine for now, since these unsupported browser logs are the only thing using it. Or, we could reason that less validation on this endpoint is better